streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamTransformValues.java

```diff
@@ -46,8 +58,104 @@ public KStreamTransformValuesProcessor(ValueTransformer<V, R> valueTransformer)
         }
 
         @Override
-        public void init(ProcessorContext context) {
-            valueTransformer.init(context);
+        public void init(final ProcessorContext context) {
```

Comment:
> About punctuate: that is a good point, and it makes me thinking if we should just change the return type from R to void then (and I know we need a KIP for that..).

I support this. It should be part of the uber KIP we plan to do for all those API changes we discussed.